app/serializers/host_serializer.rb

```diff
@@ -11,6 +11,7 @@ class HostSerializer
   end
 
   has_many :test_results
+  has_many :test_result_profiles
```

Comment:
Again, not sure why we're changing the REST API. This addition makes sense, of course, I just don't know why it should be added with this change.

Reply:
REST API changes dropped.